caffe2/python/_import_c_extension.py

```diff
@@ -10,12 +10,12 @@
 # if that still fails, we will exit loud.
 with extension_loader.DlopenGuard():
     has_hip_support = False
-    has_gpu_support = False
+    has_cuda_support = False
```

Comment:
In order to make this module itself backward compatible, we should set has_gpu_support properly in this file.
e.g. such code could exist:
```
import caffe2.python._import_c_extension.py as C
if C.has_gpu_support:
    pass
```

Reply:
@bddppq hope the latest commit works.